Rollbar/RollbarConfig.cs

```diff
@@ -0,0 +1,31 @@
+namespace RollbarDotNet
+{
+    public class RollbarConfig
+    {
+        public RollbarConfig() : this("")
+        {
+        }
+
+        public RollbarConfig(string accessToken)
+        {
+            AccessToken = accessToken;
+            Environment = "production";
+            Enabled = true;
+            LogLevel = ErrorLevel.Debug;
+            ScrubField = new[] { "passwd", "password", "secret", "confirm_password", "password_confirmation" };
```

Comment:
Yeah this needs to be improved some more, it's not being used yet, because nothing has been built in to support ASP.NET web apps yet.